client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java

```diff
@@ -170,6 +170,13 @@ public void getModel(String modelId, ActionListener<MLModel> listener) {
         client.execute(MLModelGetAction.INSTANCE, mlModelGetRequest, getMlGetModelResponseActionListener(listener));
     }
 
+    @Override
+    public void getModel(String modelId, String tenantId, ActionListener<MLModel> listener) {
```

Comment:
I don't see the tenantId being used in this function.

Reply:
thanks, added.